lib/server.js

```diff
@@ -207,6 +207,11 @@ class Session extends EventEmitter {
         state: 'open'
       }
     };
+
+    this.on('eof', () => {
```

Comment:
Might include a comment here about this being necessary to properly close the connection for some SFTP clients.

Reply:
This comment seems to be addressed